examples/README.md

```diff
@@ -0,0 +1,19 @@
+The following projects are the result of creating new projects from
+the templates:
+
+- Google.Cloud.Functions.Examples.SimpleHttpFunction
+- Google.Cloud.Functions.Examples.SimpleEventFunction
+- Google.Cloud.Functions.Examples.FSharpHttpFunction
+- Google.Cloud.Functions.Examples.FSharpEventFunction
+- Google.Cloud.Functions.Examples.VbEventFunction
```

Comment:
nit: swap this line with the following so that HttpFunction is first on all langs.

Reply:
Doh! That was absolutely the intention. Will fix now :)